apps/squeaknode/docker-compose.yml

```diff
@@ -0,0 +1,44 @@
+version: "3.7"
+
+services:
+  web:
+    image: yzernik/squeaknode:v0.1.89@sha256:249337c965855df5bd883a680ff3172993e2273415e63671e31ed4df6850ae5a
+    restart: on-failure
+    stop_grace_period: 1m
+    ports:
+      - 8994:8994
+      - 12994:12994
+      - 8555:8555
+      - 18555:18555
+    volumes:
+      - ${APP_DATA_DIR}/sqk:/sqk
+      - ${LND_DATA_DIR}:/lnd:ro
+    user: "1000:1000"
+    environment:
+      # Bitcoin Core connection details
+      SQUEAKNODE_BITCOIN_RPC_HOST: $BITCOIN_IP
+      SQUEAKNODE_BITCOIN_RPC_PORT: $BITCOIN_RPC_PORT
+      SQUEAKNODE_BITCOIN_RPC_USER: $BITCOIN_RPC_USER
+      SQUEAKNODE_BITCOIN_RPC_PASS: $BITCOIN_RPC_PASS
+
+      # LND environment variables
+      SQUEAKNODE_LND_HOST: $LND_IP
+      SQUEAKNODE_LND_RPC_PORT: $LND_GRPC_PORT
+      SQUEAKNODE_LND_TLS_CERT_PATH: "/lnd/tls.cert"
+      SQUEAKNODE_LND_MACAROON_PATH: "/lnd/data/chain/bitcoin/$BITCOIN_NETWORK/admin.macaroon"
+
+      # App specific environment variables
+      SQUEAKNODE_WEBADMIN_ENABLED: "true"
+      SQUEAKNODE_WEBADMIN_USERNAME: "umbrel"
+      SQUEAKNODE_WEBADMIN_PASSWORD: "moneyprintergobrrr"
+
+      SQUEAKNODE_CORE_NETWORK: "$BITCOIN_NETWORK"
+      SQUEAKNODE_CORE_SQK_DIR_PATH: "/sqk"
+
+      DEBUG: "squeaknode:*"
+
+  envoy:
+    image: envoyproxy/envoy:v1.16.0
+    network_mode: "host"
```

Comment:
You should expose the explicit ports you need to open instead of giving envoy host networking access here.

Reply:
Fixed.